package.json

```diff
@@ -92,17 +91,21 @@
         "webpack-dev-server": "^4.5.0"
     },
     "peerDependencies": {
-        "react": "^16.9.0",
-        "react-dom": "^16.9.0"
+        "react": "^18.2.0",
+        "react-dom": "^18.2.0"
     },
     "dependencies": {
+        "@babel/plugin-transform-runtime": "^7.23.6",
+        "@babel/runtime": "^7.23.6",
         "@fortawesome/fontawesome-free": "^5.15.2",
         "@fortawesome/fontawesome-svg-core": "~1.2.34",
         "@fortawesome/free-solid-svg-icons": "^5.15.2",
         "@fortawesome/react-fontawesome": "^0.1.14",
         "@tweakpane/core": "^1.0.8",
         "@tweakpane/plugin-essentials": "^0.1.4",
+        "@types/react-dom": "^18.2.18",
         "comlink": "^4.3.1",
+        "detect-collisions": "^9.2.2",
```

Comment:
this probably should not be a "real" dependency since it is only used by a simulator class that lives in examples, and is not a dependency of the actual simularium-viewer.  (Do we need a separate package.json for the test app to capture that dependency?)

Reply:
ideally yes.  I don't know if it's worth it but yeah we are building and bundling two projects really.  One for the library that is the viewer and the other for the test app.

Reply:
ok, I moved the scripts, the webpack file, the dev deps all into examples. I also created a `src` in examples to add a little organization, so it ends up looking like a lot of file changes